slidedeck/src/slides/09-special-topics/10-testing.pug

```diff
@@ -0,0 +1,70 @@
+section
+  h2 Advanced Testing
+
+section
+  h3 Don't try to test library code!
+  p Third-party libs like Redux and React-Router are already well-tested
+  p Focus on *your* code
+
+section
+  h3 A very simple component can be wrapped in many layers
+  script(type="text/plain", class="language-jsx").
+    const MyAwesomeComponent = (props) => ...;
+    const mapStateToProps = ...;
+    const mapDispatchToProps = ...;
+    export default withRouter(connect(mapStateToProps, mapDispatchToProps)(MyAwesomeComponent));
+
+  script(type="text/plain", class="language-jsx").
+    const wrapper = mount(
+      <Provider store={mockStore}>
+        <MemoryRouter>
+          <MyAwesomeComponent />
+        </MemoryRouter>
+      </Provider>
+    );
+
+section
+  h3 Peeling those layers back can make testing a lot simpler
+  script(type="text/plain", class="language-jsx").
+    export const _MyAwesomeComponent = (props) => ...;
```

Comment:
Why the underscore here?

Reply:
Was trying to figure out a way to more effectively manage the "named vs default" export issue when testing wrapped/unwrapped versions of components